webplugin/js/app/kommunicate-ui.js

```diff
@@ -1178,6 +1178,45 @@ KommunicateUI = {
                     'vis'
                 );   
             }
+            var restartConversation = document.getElementById(
+                'mck-restart-conversation'
+            );
+            restartConversation.addEventListener('click', function () {
+                Kommunicate.startConversation(); 
+            })
+            document.getElementById('mck-submit-comment').disabled = false;
+            kommunicateCommons.modifyClassList(
+                { class: ['mck-rating-box'] },
+                '',
+                'selected'
+            );
+            kommunicateCommons.modifyClassList(
+                {
+                    class: ['mck-box-form'],
+                },
+                'n-vis',
+                'vis'
+            );
+            kommunicateCommons.modifyClassList(
+                {
+                    class: ['mck-csat-text-1'],
+                },
+                '',
+                'n-vis'
+            );
+            kommunicateCommons.modifyClassList(
+                {
+                    id: ['mck-sidebox-ft'],
+                },
+                'km-mid-conv-csat'
+            );
+            kommunicateCommons.modifyClassList(
+                {
+                    id: ['csat-1'],
+                },
+                'vis',
+                'n-vis'
+            );
```

Comment:
do we want to do this change for all the conversations? both normal and zendesk chat one?
@pranavp99

Reply:
if no, lets try to push this code to zendesk chat related code file only.

Reply:
no we are only doing it for zendesk, there's a checker if(convResolvedFromZendesk)